model/task_history.go

```diff
@@ -389,35 +391,20 @@ func (self *taskHistoryIterator) GetDistinctTestNames(ctx context.Context, env e
 			{"$unwind": fmt.Sprintf("$%v", testResultsKey)},
 			{"$group": bson.M{"_id": fmt.Sprintf("$%v.%v", testResultsKey, task.TestResultTestFileKey)}},
 		},
-		opts,
 	)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting cursor")
-	}
-	if cursor == nil {
-		return nil, errors.New("nil cursor returned")
-	}
-	defer cursor.Close(ctx)
-
-	names := []string{}
-	for cursor.Next(ctx) {
-		select {
-		case <-ctx.Done():
-			return nil, errors.New("context cancelled")
-		default:
-			res := bson.M{}
-			if err := cursor.Decode(&res); err != nil {
-				return nil, errors.Wrapf(err, "error decoding result")
-			}
-			// remove quotes from value
-			val := strings.Replace(res["_id"].(string), "\"", "", -1)
```

Comment:
Is this string replacement not necessary anymore?

Reply:
I'm not sure. It was added in #4500. When I reverted that commit this went away too. I didn't put it back because although it wasn't clear what necessitated it there it was clear that it was [a result of the changes there](https://github.com/evergreen-ci/evergreen/pull/4500/files#r597719059).

Reply:
This was just a legitimate issue with how MongoDB was returning the test results; they were all being printed as 
` "\"TestName"\"` or something when I checked with staging. Testing in staging (when you can haha) will determine if this is still necessary.

Reply:
I did eventually get to test this in staging. It was a little tricky because we already switched test results over to cedar and the query is limited to the last 10 commits, but it worked out by matching on a date in the past.
It returned this for the `test-service` task:
```
[
  "TestActivateVersion",
  "TestAssignNextAvailableTaskWithDispatcherSettingsVersionLegacy",
  "TestAssignNextAvailableTaskWithDispatcherSettingsVersionTunable",
  "TestCedar",
  "TestCheckHostHealth",
  "TestCheckHostWrapper",
  "TestConsistentTaskAssignment",
  "TestGetBuildInfo",
  "TestGetBuildStatus",
  "TestGetDisplayTaskInfo",
  "TestGetDockerfile",
  "TestGetHostDNS",
  "TestGetHostFromCache",
  "TestGetRecentVersions",
  "TestGetTaskInfo",
  "TestGetTaskStatus",
  "TestGetVariantsWithCommand",
  "TestGetVersionInfo",
  "TestGetVersionInfoViaRevision",
  "TestGetVersionStatus",
  "TestModifyHostStatusWithUpdateStatus",
  "TestNextTask",
  "TestPatchListModulesEndPoints",
  "TestProjectRoutes",
  "TestServiceStatusEndPoints",
  "TestStuckHostEndpoints",
  "TestTaskHistory",
  "TestTaskHistory/TestGetVersionsInWindow",
  "TestTaskHistory/TestSurroundingVersions",
  "TestTaskHistory/TestSurroundingVersionsSort",
  "TestTaskLifecycleEndpoints",
  "TestValidateTaskEndDetails",
  "TestVerifyAliasExists"
]
```
which is what I'd expect.